### DISCUSSION TITLE
How to use batch processing with sdxl "inpainting" and "multicontrolnet" and (Another question) Is there anyway to use them together (regardless of batch processing)

### DISCUSSION THREAD

**Discussion by loliana232** [huggingface/diffusers#6979] [Q&A] [answered]
Hello, I have 2 questions 
1. Anyone know how to use batch processing for inpainting and multicontrolnet ???
2. And is there anyway to use inpainting and multicontrolnet as together at  the same time ??!!

**Comment by sayakpaul** [comment, marked as answer]
Refer to https://huggingface.co/docs/diffusers/main/en/using-diffusers/controlnet. 

Batch processing can be enabled by using the `num_images_per_prompt` argument.
